index.js

```diff
@@ -1,6 +1,80 @@
 /* jshint node: true */
 'use strict';
 
+const Funnel = require('broccoli-funnel');
+const chalk  = require('chalk');
+
+// We import these to merge the instrumentation options into the base classes
+// we do this instead of overriding the core commands because other high profile
+// addons such as ember-cordova, ember-cli-nwjs, and ember-electron have had to override
+const serveCommand = require('ember-cli/lib/commands/serve');
+const buildCommand = require('ember-cli/lib/commands/build');
+const testCommand = require('ember-cli/lib/commands/test');
+
+function patchCommand(command, newOption) {
+  command.prototype.availableOptions.push(newOption);
```

Comment:
We should check for it before pushing, that allows this to become a polyfill if we ever make this an upstream option.